docs/4.1/admin-guide.md

```diff
@@ -2223,6 +2228,56 @@ The `license_file` path can be either absolute or relative to the configured
     Only Auth servers require the license. Proxies and Nodes that do not also
     have Auth role enabled do not need the license.
 
+## Backing Up Teleport
```

Comment:
The way I read this section, I think it might give someone the impression that the `get all`/`--bootstrap` pattern is the way normal backups should be performed, which isn't the case.  Normal backups should still be done by backing up the filesystem/backend directly.  The `get all`/`--bootstrap` pattern is more targeted at use-cases like migrating between backends, or setting up test/example clusters.

Maybe we could title this section "Backing up Resources" or "Backing up Dynamic Configuration"?

Reply:
Thanks @fspmarshall I've updated it.